README.md

```diff
@@ -127,27 +127,6 @@ before you can use the Composer scripts.
 You can run `composer` (or `ddev composer`) to display a list of all available
 Composer commands and scripts.
 
-| Script name            | Action                        | Tool                                                                            |
```

Comment:
should we also mention that all scripts descriptions are in composer.json in script-description section?

Reply:
Done.